README.md

```diff
@@ -1,74 +1,270 @@
+
+`flex status`
+
+   Displays the health of the current (the one you initiated last) Flex Service. To get the status of a different service, specify its service ID using the `--serviceId` option.
+
+`flex list`
+
+  Lists all Flex Services for a domain (app or organization), excluding external Flex Services. Specify domain using `--domain` and then an app or organization ID using `--id`. If you skip the domain option, the command lists the services inside the domain you've configured as part of running `flex init`.
+
+`flex logs`
+
+   Retrieves and displays Flex Services logs. Logs calls return 100 entries by default and can return up to 2,000 entries. Logs are displayed in the following format: `<runtime id> <timestamp> - <message>`. Run `kinvey flex logs -h` for advanced `logs` command usage details, including returning more than the default number of entries, specifying a page number, or filtering by timestamp. Log for external Flex Services are not returned.
+  
+`flex recycle`
+   
+   Recycles the current (the one you initiated last) Flex Service. To recycle a different service, specify its service ID using the `--serviceId` option.
+
+`flex delete`
+
+   Deletes the current Flex Service configuration from the Node.js project directory that it has been executed in.
+
+`help`
+
+   Prints general usage instructions. For detailed command usage instruction, use the `--help` option with the command.
+
+## Options
+
+`--domain <app|org>`
+
+   Specifies the domain type for commands like `flex list`.
+
+`--email <e-mail>`
+
+   Email address of your Kinvey account.
+
+`--host <host>`
+
+   Hostname of a Kinvey instance. It has a default value of `https://manage.kinvey.com/` which most customers should use. If you are a customer on a dedicated Kinvey instance, enter your dedicated host name as either an absolute URI (`https://kvy-us2-manage.kinvey.com/`) or an instance name only (`kvy-us2`).
+
+`--id <app or organization ID>`
+
+   App or organization ID for use with `--domain <app|org>`.
+
+`--password <password>`
+
+   Password for your Kinvey account.
+
+`--profile <profile>`
+
+   Profile to use.
+
+`--serviceId <Flex Service ID>`
+
+   Specifies a Flex Service, e.g. for use with the `flex status` command.
+
+`--silent`
+
+   Suppresses any output. Useful for scripting.
+
+`--suppress-version-check`
+
+   Prevents Kinvey CLI to check for new versions, which normally happens each execution.
+   
+`--verbose`
+
+   Prints additional debug messages.
+
+`--version`
+
+   Prints the version number of `kinvey-cli`.
+
+`--help, -h`
+
+   When used after a `kinvey-cli` command, shows its usage instructions.
+
+## Environment Variables
+
+Use these environment variables to specify profile information for the `profile create` command when you don't want to specify it on the command line. Keep in mind that any values specified at he command line take precedence over the environment variable values.
```

Comment:
Missing 't' in 'the': 'specified at he command line'

Reply:
👍